utils/EA/fitness.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.model_selection import cross_val_score
+
+from .population import phenotype
+
+def fitness(sick_data, sick_labels, healthy_data, healthy_labels):
+    def fitness_(indiv):
+        pheno = phenotype(indiv)
+        # select features and only pass this data to evaluate
+
+        return evaluate(sick_data[:, pheno], sick_labels, healthy_data[:, pheno], healthy_labels)
+    return fitness_
+
+def evaluate(sick_data, sick_labels, healthy_data, healthy_labels):
+    clf = DecisionTreeClassifier()
+
+    if sick_data.shape[1] > 10:
```

Comment:
Could we move this into a constant that is a little easier to understand? Makes changes easier :)
Also Lukas proposed to return Double.minValue, and I second that

Reply:
Definitely! I just did this to decrease the runtime. You can imagine how long it could take if you also train classifiers with more than 10 features :-D
I also had a version which penalizes the number of features as we try to follow Occams Razor Principle (take as few features as possible). But my function did not work well and I mostly got only 1 feature...